packages/web/hooks/fiat-getters.ts

```diff
@@ -0,0 +1,71 @@
+import { Dec, DecUtils, Int, PricePretty } from "@keplr-wallet/unit";
+import { type Asset, DEFAULT_VS_CURRENCY } from "@osmosis-labs/server";
+
+// getTokenInFeeAmountFiatValue returns the fiat value of the token in fee amount.
+// If any of the input values are undefined, it returns a PricePretty with 0 value.
+// The client must handle the case where the returned value is 0.
+// Returns zero on any truncation
+// - asset used to determine the precision exponent
+// - tokenInFeeAmount the fee amount in the token in asset (scaled according to the asset's precision)
+// - inAmountInputPrice the price of the token in asset in fiat (scaled according to the asset's precision)
+export function getTokenInFeeAmountFiatValue(
+  asset: Asset | undefined,
```

Comment:
Also I think it's best the `Currency` (from "@osmosis-labs/types") type be used instead of `Asset`, it's a bit less constrictive and `Asset` conforms to `Currency`.

Reply:
[9800e8b](https://github.com/osmosis-labs/osmosis-frontend/pull/3070/commits/9800e8ba53db4e33984ef04c486224f5ab2486fc)